tests/Backend/ExternalBuckets/SnowflakeBYODBTest.php

```diff
@@ -80,21 +91,33 @@ public function testRegisterExternalBucketInBYODBEnvironment(): void
 
         $this->_client->registerBucket(
             'test-bucket-registration',
-            [self::TESTDB, self::TESTSCHEMA],
+            [self::TEST_DB, self::TEST_SCHEMA],
             'in',
             'will not fail',
             'snowflake',
             'test-bucket-registration',
         );
 
-        $tables = $this->_client->listTables('in.test-bucket-registration');
+        $tables = $this->_client->listTables($bucketId);
+        $this->assertCount(2, $tables);
+
+        $db->executeQuery(
+            sprintf(
+                'DROP VIEW %s;',
+                SnowflakeQuote::quoteSingleIdentifier(self::TEST_VIEW),
+            ),
+        );
+
+        $this->_client->refreshBucket($bucketId);
+
+        $tables = $this->_client->listTables($bucketId);
         $this->assertCount(1, $tables);
-        $this->_client->refreshBucket('in.test-bucket-registration');
 
+        $this->_client->dropBucket($bucketId);
```

Comment:
todle je realna zmena, zbytek je omacka, co jsem si neodpustil, kdyz jsem to videl

Reply:
Toto tu asi nemusi byt, maze sa to na zaciatku testu, ale nicomu to nevadi